src/Tribe/Views/V2/View.php

```diff
@@ -1352,6 +1355,7 @@ public function has_next_event( array $events, $overwrite_flag = true ) {
 	protected function setup_template_vars() {
 		if ( empty( $this->repository_args ) ) {
 			$this->repository_args = $this->filter_repository_args( $this->setup_repository_args() );
+			$this->repository = tribe_events();
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 6 spaces but found 1 space

<sub>hash: dec4ee89defabdb9145aea858c671db5</sub>